store/mongo/datastore_mongo.go

```diff
@@ -1884,7 +1886,11 @@ func (db *DataStoreMongo) DeleteDeviceDeploymentsHistory(ctx context.Context,
 		return err
 	}
 
-	return nil
+	database = db.client.Database(DatabaseName)
+	collDevs = database.Collection(CollectionDevicesLastStatus)
+	_, err := collDevs.DeleteMany(ctx, bson.M{StorageKeyDeviceDeploymentDeviceId: deviceID})
```

Comment:
in the first place I'm not sure what we should do here; maybe deleting the document is OK
question - why DeleteMany? this collection can contain single document per device (we use device id as id)

Reply:
I just chose to delete -- since we do not need successful statuses, why waste space? delete many -- just in case, no hard reasons. if you feel we should keep the records, let me know.

Reply:
I'm OK with keeping it this way